packages/compiler-cli/src/ngtsc/typecheck/extended/index.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {ErrorCode} from '../../diagnostics';
+import {TemplateCheck} from './api/api';
```

Comment:
When you have a separate package, always import from the index:
```suggestion
import {TemplateCheck} from './api';
```